### PR TITLE
Remove .bsp folder

### PR DESCRIPTION
**Pull request by mgenereu**
Remove .bsp folder which is local specific.  Also added to .gitignore to not commit in the future.

This existing folder/file conflicted with IntelliJ starting up a BSP project.

### CODE REVIEW

File: .bsp/sbt.json [hunk 1]
```diff
@@ -1 +0,0 @@
-{"name":"sbt","version":"1.4.2","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["C:\\Users\\John\\scoop\\apps\\oraclejdk\\current/bin/java","-Xms100m","-Xmx100m","-classpath","C:\\Users\\John\\scoop\\apps\\sbt\\current\\bin\\sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
```
Comment (mgenereu) [inline .bsp/sbt.json:1]: You can see it's encoded to @jdegoes personal java binary location.

**Comment by anovakovic01** [comment]
@mgenereu While you're at it, maybe you could remove `.vscode/` too.

**Comment by mgenereu** [comment]
> @mgenereu While you're at it, maybe you could remove `.vscode/` too.

I looked at the file in there and it looked like "project settings" as opposed to "local settings".  Take a look and tell me what you think.

**Comment by anovakovic01** [comment]
I agree that `.bsp` was tied more to the local setup, but `.vscode` is tied to and generated by the specific editor (VS Code), so I don't think that it needs to be present in the repository.

**Comment by mgenereu** [comment]
> I agree that `.bsp` was tied more to the local setup, but `.vscode` is tied to and generated by the specific editor (VS Code), so I don't think that it needs to be present in the repository.

Done.

**Review by anovakovic01 — APPROVED** [review]

**Review by jdegoes — APPROVED** [review]

**Pull request merged**